src/components/molecules/AccountTeaser.module.css

```diff
@@ -0,0 +1,41 @@
+.blockies {
+  aspect-ratio: 1/1;
+  width: 13%;
+  height: 13%;
+  border-radius: 50%;
+  margin-left: 0;
+  margin-right: calc(var(--spacer) / 4);
+}
+
+.teaser {
+  max-width: 40rem;
+  height: 100%;
+}
+
+.link {
+  composes: box from '../atoms/Box.module.css';
+  padding: calc(var(--spacer) / 2) !important;
+  font-size: var(--font-size-mini);
+  height: 90%;
+  color: var(--color-secondary);
+  position: relative;
+}
+
+.link span {
+  font-size: var(--font-size-large);
+  margin-right: calc(var(--spacer) / 3);
+  margin-top: calc(var(--spacer) / 4);
```

Comment:
I'll suggest removing this margin and rather use flex on parent component (`align-items:center`) for centring the child components